### PR TITLE
Some Fixes

### PR DESCRIPTION
**Pull request by daneoshiga**
That's a pretty cool post :)

Just some fixes from what I got while reading it.

### CODE REVIEW

File: _posts/2020-05-23-part1-iam-oauth-provider.markdown [hunk 1]
```diff
@@ -103,7 +100,7 @@ That’ll create a directory `users`, which is laid out like this:
     └── views.py
 {% endhighlight %}
 
-If you’re starting a new project, it’s highly recommended to set up a custom user model, even if the default [User] model is sufficient for you. This model behaves identically to the default user model, but you’ll be able to customize it in the future if the need arises.
+If you’re starting a new project, it’s highly recommended setting up a custom user model, even if the default [User] model is sufficient for you. This model behaves identically to the default user model, but you’ll be able to customize it in the future if the need arises.
```
Comment (wiliamsouza) [inline _posts/2020-05-23-part1-iam-oauth-provider.markdown]: This is a citation from Django doc maybe this should be as is?

**Comment by wiliamsouza** [comment]
LGTM

File: _posts/2020-05-23-part1-iam-oauth-provider.markdown (hunk 1 above)
Comment (daneoshiga) [reply _posts/2020-05-23-part1-iam-oauth-provider.markdown]: makes sense, reverted.

**Comment by wiliamsouza** [comment]
Thanks :heart:

**Pull request merged**